.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+tmp
```

Comment:
I've substituted  `afterEach` with `beforeEach` in `test.js`:
```diff
- afterEach(function () {
-	try {
-		fs.unlinkSync('tmp');
-	} catch (err) {}
+beforeEach(function () {
```
This allows one to run `mocha --bail` and examine the files created by a failing test. So I think ignoring them would be useful.